stl/inc/mdspan

```diff
@@ -227,7 +227,7 @@ public:
         : extents(_Extents_from_tuple{}, _STD tie(_Exts...), make_index_sequence<rank_dynamic()>{}) {
 #if _ITERATOR_DEBUG_LEVEL != 0
         auto _Check_extent = []<class _Ty>(const _Ty& _Ext) {
-            if constexpr (_Is_standard_integer<_Ty>) {
+            if constexpr (_Is_standard_integer<remove_cv_t<_Ty>>) {

```

Comment:
This change shouldn't be necessary. `_OtherIndexTypes... _Exts` are being taken by value and their types are deduced, so they can't be cv-qualified. Below, `!same_as<_Ty, bool>` already assumes that we don't have to worry about const.